samlauthenticator/samlauthenticator.py

```diff
@@ -296,6 +296,25 @@ class SAMLAuthenticator(Authenticator):
         a non-zero status in the failure case.
         '''
     )
+    xpath_role_location = Unicode(
+        default_value='//saml:Role/text()',
```

Comment:
I'm not too familiar with the SAML Spec, but I don't know that I've ever seen a `Role` field come through in an assertion. Where did you get this XPath? If you haven't seen this in the field, it might be better to leave this XPath as `None`. I think that's a safer practice.

Reply:
I made up this value. Makes sense to default to `None` if not supplied.